### DISCUSSION TITLE
The relationship between location and point?

### DISCUSSION THREAD

**Discussion by HuJunnj** [openTCS/opentcs#121] [General]
I want to know that the relationship between location and point? and we who to used the location element? thanks very much if you can tell me these answers to problems.

**Comment by swltr** [comment]
@HuJunnj I recommend to read the documentation about this.

**Comment by freekevinCode** [comment]
A set of links to points that the location can be reached from. To be of any use for vehicles in the plant model, a location needs to be linked to at least one point.  --from the original documentation
